docs/configuration.md

```diff
@@ -71,6 +71,8 @@ Iceberg tables support table properties to configure table behavior, like the de
 | write.delete.target-file-size-bytes| 67108864 (64 MB)   | Controls the size of delete files generated to target about this many bytes |
 | write.distribution-mode            | none               | Defines distribution of write data: __none__: don't shuffle rows; __hash__: hash distribute by partition key ; __range__: range distribute by partition key or sort key if table has an SortOrder |
 | write.delete.distribution-mode     | hash               | Defines distribution of write delete data          |
+| write.update.distribution-mode     | hash               | Defines distribution of write update data          |
+| write.merge.distribution-mode      | none               | Defines distribution of write merge data           |
```

Comment:
Looks like the default here is a bit complicated.  From my reading of the code, it picks 'write.distribution-mode' if it is set, else "range" if sorted, "none" if unsorted.  Do you see that as well, and should we mention that?

I feel we also should add something to spark-writes about distribution mode.